llvm/tools/llvm-exegesis/llvm-exegesis.cpp

```diff
@@ -262,6 +262,12 @@ static cl::opt<BenchmarkRunner::ExecutionModeE> ExecutionMode(
                           "allows for the use of memory annotations")),
     cl::init(BenchmarkRunner::ExecutionModeE::InProcess));
 
+static cl::opt<unsigned>
+    BenchmarkRepeatCount("benchmark-repeat-count",
+                         cl::desc("The number of times to repeat the benchmark "
```

Comment:
I would phrase this as "number of times to repeat measurements on the benchmark before aggregating the results", to make the difference with `-num-repetitions` clearer.

Reply:
Good point. Fixed in the latest revision.